### DISCUSSION TITLE
联 机 功 能 的 大 坑

### DISCUSSION THREAD

**Discussion by Yue-cn** [Meloong-Git/PCL#2595] [讨论]
你发现了嘛？为什么**网易**我的世界 没有兼容到 1.7.10 以下版本？
你可能会想到:
* ~懒得写Mod~
* ~Mod 太少~

这些都不是关键嘛，由于Minecraft 在 1.7 开始采用 Netty **重写**了联机部分的 Code，
导致与之前大有不同，现在包的第一位一般是 0X10 或 0X0F

还有，现在的 MOTD 只允许在 [AD] 属性中传入端口号，而早期版本是可以直接传入完整的链接。
![image](https://github.com/Hex-Dragon/PCL2/assets/79188080/42a2eb78-191e-44d8-8b58-8e449132e5a1)

如果换到现今的版本，这种老方法就用不了咯，主要是通过 **局域网 Games** 列表加入无需经过正版验证，除非你代理了个什么 Minecraft Server（别问我，之前整过整个老活)

~（再说一句，怎么 BUGJANG 的 PingPong 是一次性的啊）~

再有吧，现在一般的方案是采用外置的方式。（保守派）

**Comment by SqrtTan** [comment]
还有个原因是因为bugjump1.7不支持中文……
